lib/mix/tasks/nerves_hub.device.ex

```diff
@@ -25,13 +25,14 @@ defmodule Mix.Tasks.NervesHub.Device do
     * `--tag` - (Optional) Multiple tags can be set by passing this key multiple
       times.
 
-  ## provision
+  ## burn
 
-  Provision a NervesHub device. This requires that the device was already created.
-  Calling provision without passing command line options will generate a new cert
-  pair for the device. The command will end with calling mix firmware.burn.
+  Burn provisioned firmware to a NervesHub device. This requires that the device 
```

Comment:
Instead of the first sentence, maybe something along the lines of "Combine a firmware image with NervesHub provisioning information and burn the result to an attached MicroSD card or file."